components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/impl/CertificatesApiServiceImpl.java

```diff
@@ -0,0 +1,282 @@
+package org.wso2.carbon.apimgt.rest.api.publisher.impl;
```

Comment:
Missing license header

Reply:
Added the license header https://github.com/wso2/carbon-apimgt/pull/5564/commits/807e82f1f02e867afd778dc1b4cbe40e15325e7a